samples/bidirectional-chat/csharp/Function.cs

```diff
@@ -0,0 +1,124 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Azure.WebJobs.Extensions.SignalRService;
+using Microsoft.AspNetCore.SignalR;
+
+namespace bidirectional_chat
+{
+    public class SimpleChat : ServerlessHub
+    {
+        private const string Hub = "simplechat";
```

Comment:
>"simplechat" [](start = 35, length = 12)

use GetType().Name to uniform the method of getting hub value?

Reply:
Yes!